### PR TITLE
Issue forms for docs opensource

### PR DESCRIPTION
**Pull request by ramyaparimi**
### Why:

Issue form for docs open source
Related to https://github.com/github/docs/pull/8565

### What's being changed:

rendered content!!

[improve existing docs](https://github.com/github/docs/blob/partner-program-issue-form/.github/ISSUE_TEMPLATE/improve-existing-docs.yaml)
[improve the site](https://github.com/github/docs/blob/partner-program-issue-form/.github/ISSUE_TEMPLATE/improve-the-site.yml)
[partner program](https://github.com/github/docs/blob/partner-program-issue-form/.github/ISSUE_TEMPLATE/partner-contributed-documentation.yml)

### Check off the following:

- [x] I have reviewed my changes in staging (look for the latest deployment event in your pull request's timeline, then click **View deployment**).
- [ ] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Review by heiskr — DISMISSED** [review]
Makes sense from an engineering perspective, make sure to get @chiedo and @janiceilene to review before merging.

File: .github/ISSUE_TEMPLATE/improve-existing-docs.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+title: '[What I really want the title to be]'
+labels:
+  - content
+body:
+  - type: markdown
+    attributes:
+      value: |
+        **HUBBERS!! THIS IS GITHUB/DOCS OPEN SOURCE REPO. OPEN AN ISSUE IN GITHUB/DOCS-CONTENT INSTEAD.**
+
+        * For questions, ask in [Discussions](https://github.com/github/docs/discussions)
```
Comment (janiceilene) [inline .github/ISSUE_TEMPLATE/improve-existing-docs.yaml:12]: Can you go through and add a `.` to all of the complete sentences? We use periods at the end of any list item that's a complete sentence (like these ones), there's some more details in the [style guide](https://slack.com/intl/en-nl/help/articles/204475027-Deactivate-a-members-account) ✨

File: .github/ISSUE_TEMPLATE/improve-the-site.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
+name: Improve the docs.github.com site
+description: Make a suggestion or report a problem about the technical implementation of docs.github.com.
+title: "'[What I really want the title to be]': "
+labels:
+  - content
+body:
+  - type: markdown
+    attributes:
+      value: |
+        **HUBBERS!! THIS IS GITHUB/DOCS OPEN SOURCE REPO. OPEN AN ISSUE IN GITHUB/DOCS-CONTENT INSTEAD.**
```
Comment (janiceilene) [inline .github/ISSUE_TEMPLATE/improve-the-site.yml:10]: Any changes you make to this line or additional `.` you add to the template above ☝️ should be added to this one too, to keep them matchy matchy 🤝

File: .github/ISSUE_TEMPLATE/partner-contributed-documentation.yml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+          required: true
+        - label: MUST pass the automated CI checks
+          required: true
+        - label: MUST include links to supporting material demonstrating the functionality being documented (this can be a link to a public GitHub repo, _or_ a video / screencast walkthrough)
+          required: true
+
+  - type: markdown
+    attributes:
+      value: |
+        * Once all tasks are completed, please mention `@github/docs-content` for next steps.
```
Comment (janiceilene) [inline .github/ISSUE_TEMPLATE/partner-contributed-documentation.yml:59]: Can we remove the bullet point on this line and the next one?

**Review by janiceilene — DISMISSED** [review]
This is looking great! I love all the little required `*` 😭 Hopefully it'll make our lives so much better! I left a few comments and suggestions for you to consider ⚡

File: .github/ISSUE_TEMPLATE/improve-existing-docs.yaml [hunk 4]
```diff
@@ -0,0 +1,49 @@
+name: Improve existing content
+description: Make a suggestion to improve the content in an existing article
+title: '[What I really want the title to be]'
+labels:
+  - content
+body:
+  - type: markdown
+    attributes:
+      value: |
+        **HUBBERS!! THIS IS GITHUB/DOCS OPEN SOURCE REPO. OPEN AN ISSUE IN GITHUB/DOCS-CONTENT INSTEAD.**
```
[the remark this answers is not in the document]
Comment (ramyaparimi) [reply .github/ISSUE_TEMPLATE/improve-existing-docs.yaml:10]: @lee-dohm fixed this in the other [PR](https://github.com/github/docs/pull/8565) I opened for specifically for only OS contributors.
This is how it looks [now](https://github.com/github/docs/issues/new?assignees=&labels=content&template=improve-existing-docs.yaml)

File: .github/ISSUE_TEMPLATE/improve-existing-docs.yaml (hunk 1 above)
Comment (ramyaparimi) [reply .github/ISSUE_TEMPLATE/improve-existing-docs.yaml:12]: I am going to add `.` to the list points 🐬 Should I open another PR or can I just make the changes in this PR?

File: .github/ISSUE_TEMPLATE/improve-existing-docs.yaml (hunk 4 above)
[the remark this answers is not in the document]
Comment (ramyaparimi) [reply .github/ISSUE_TEMPLATE/improve-existing-docs.yaml:10]: @lee-dohm fixed this in the other [PR](https://github.com/github/docs/pull/8565) I opened for specifically for only OS contributors.
This is how it looks [now](https://github.com/github/docs/issues/new?assignees=&labels=content&template=improve-existing-docs.yaml)

File: .github/ISSUE_TEMPLATE/improve-existing-docs.yaml [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+      description: By submitting this issue, you agree to follow our [Code of Conduct](https://example.com)
+      options:
+        - label: I agree to follow this project's Code of Conduct
+          required: true
+
+  - type: textarea
+    attributes:
+      label: What article on docs.github.com is affected?
+      description: |
+        Please link to the article you'd like to see updated
```
[the remark this answers is not in the document]
Comment (ramyaparimi) [reply .github/ISSUE_TEMPLATE/improve-existing-docs.yaml:29]: I am going to add . to the list points 🐬 Should I open another PR or can I just make the changes in this PR?

File: .github/ISSUE_TEMPLATE/improve-existing-docs.yaml (hunk 4 above)
[the remark this answers is not in the document]
Comment (ramyaparimi) [reply .github/ISSUE_TEMPLATE/improve-existing-docs.yaml:10]: I just noticed that issue form for improving docs site still have the same style so I fixed that one now :)

File: .github/ISSUE_TEMPLATE/partner-contributed-documentation.yml (hunk 3 above)
Comment (ramyaparimi) [reply .github/ISSUE_TEMPLATE/partner-contributed-documentation.yml]: Here is the [rendered link](https://github.com/github/docs/tree/partner-program-issue-form/.github/ISSUE_TEMPLATE) with all the changes ✨

**Comment by ramyaparimi** [comment]
I am glad `merge is blocked`! I was trying to resolve only merge conflicts 😆

**Comment by ramyaparimi** [comment]
rendered content!!

[improve existing docs](https://github.com/github/docs/blob/partner-program-issue-form/.github/ISSUE_TEMPLATE/improve-existing-docs.yaml)
[improve the site](https://github.com/github/docs/blob/partner-program-issue-form/.github/ISSUE_TEMPLATE/improve-the-site.yml)
[partner program](https://github.com/github/docs/blob/partner-program-issue-form/.github/ISSUE_TEMPLATE/partner-contributed-documentation.yml)

File: .github/ISSUE_TEMPLATE/improve-the-site.yml [hunk 6]
```diff
@@ -1,30 +1,31 @@
 name: Improve the docs.github.com site
 description: Make a suggestion or report a problem about the technical implementation of docs.github.com.
+title: "'[What I really want the title to be]': "
```
Comment (janiceilene) [inline .github/ISSUE_TEMPLATE/improve-the-site.yml:3]: Is this supposed to be here or was it a placeholder? It's in all three files.

**Review by janiceilene — DISMISSED** [review]
I left one comment for you to 👀 but other than that, I think these are ready to merge 🎉 They look great and I'm so excited to see whether or not they reduce issue spam!

File: .github/ISSUE_TEMPLATE/improve-the-site.yml (hunk 6 above)
Comment (ramyaparimi) [reply .github/ISSUE_TEMPLATE/improve-the-site.yml:3]: Ack! This must have come back in while resolving merge conflicts. Deleting them now!

**Review by lee-dohm — DISMISSED** [review]
🚢

**Review by lee-dohm — APPROVED** [review]
🚢

**Review by 9315398773 — COMMENTED** [review]
Looking good

**Pull request merged**